clientv2/linux/data_change_api.go

```diff
@@ -136,6 +138,8 @@ type DeleteDSL interface {
 
 	// VppInterface adds a request to delete an existing VPP network interface.
 	VppInterface(ifaceName string) DeleteDSL
+	// VppSpan adds VPP span to the Delete request.
+	VppSpan(span *vpp_interfaces.Span) DeleteDSL
```

Comment:
Just call it Span across the client - no need to prefix with "Vpp" since there is no LinuxSpan